nipype/interfaces/base/core.py

```diff
@@ -471,7 +480,8 @@ def run(self, **inputs):
             env['DISPLAY'] = config.get_display()
 
         runtime = Bunch(
-            cwd=os.getcwd(),
+            cwd=cwd,
+            prevcwd=syscwd,
```

Comment:
I couldn't find a place where this is being used

Reply:
Oh, forgot to add it to the report.rst